test/integration/select_test.lua

```diff
@@ -1690,3 +1690,65 @@ pgroup.test_storage_nil_err_is_processed = function(g)
     t.assert_str_contains(err.str, 'StorageError', 'Storage error class is preserved')
     t.assert_str_contains(err.str, 'My storage error', 'Storage error msg is preserved')
 end
+
+pgroup.before_test('test_storage_uninit_select_error_text', function(g)
+    helpers.call_on_storages(g.cluster, function(server)
+        server.net_box:eval([[
+            local _crud = rawget(_G, '_crud')
+            rawset(_G, '_real_crud_select', _crud.select_on_storage)
+            _crud.select_on_storage = nil
+            rawset(_G, '_real_crud_get', _crud.get_on_storage)
+            _crud.get_on_storage = nil
+        ]])
+    end)
+end)
+
+pgroup.after_test('test_storage_uninit_select_error_text', function(g)
+    helpers.call_on_storages(g.cluster, function(server)
+        server.net_box:eval([[
+            local _crud = rawget(_G, '_crud')
+            _crud.select_on_storage = rawget(_G, '_real_crud_select')
+            rawset(_G, '_real_crud_select', nil)
+            _crud.get_on_storage = rawget(_G, '_real_crud_get')
```

Comment:
We don't need to unset `_get` here after separating test cases, aren't we?

Reply:
Sure.